### PR TITLE
Lucky Number in python

### PR DESCRIPTION
**Pull request by SDakshit**
**Fixes issue:**
 Issue #4594 

**Changes:**
 Added code for finding out lucky number in python

### CODE REVIEW

File: code/mathematical_algorithms/src/lucky_number/Lucky_Number.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
```
Comment (arnavb) [inline code/mathematical_algorithms/src/lucky_number/Lucky_Number.py]: Filename should be all lowercase. Use black to reformat this file.

File: code/mathematical_algorithms/src/lucky_number/Lucky_Number.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Mar 12 21:02:07 2020
+
+@author: hp
+"""
+
+
+def isLucky(n):
```
Comment (arnavb) [inline code/mathematical_algorithms/src/lucky_number/Lucky_Number.py]: Use underscore_case.

File: code/mathematical_algorithms/src/lucky_number/Lucky_Number.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+	if isLucky.counter > n: 
+		return 1
+	if n % isLucky.counter == 0: 
+		return 0
+	
+
+	next_position = next_position - next_position / isLucky.counter 
+	
+	isLucky.counter = isLucky.counter + 1
+
```
Comment (arnavb) [inline code/mathematical_algorithms/src/lucky_number/Lucky_Number.py]: Remove unnecessary blank lines.

File: code/mathematical_algorithms/src/lucky_number/Lucky_Number.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+	
+
+	next_position = next_position - next_position / isLucky.counter 
+	
+	isLucky.counter = isLucky.counter + 1
+	
+	return isLucky(next_position) 
+	
+	
+isLucky.counter = 2
```
Comment (arnavb) [inline code/mathematical_algorithms/src/lucky_number/Lucky_Number.py]: Wrap all of this code in a `if __name__ == '__main__':`.

**Comment by arnavb** [comment]
This code was directly copied from [here](https://www.geeksforgeeks.org/lucky-numbers/). This is plagiarism, which [I feel strongly against](https://github.com/OpenGenus/cosmos/issues/4721).

Please do not copy from others, as I will lose trust in you.

**Comment by SDakshit** [comment]
Sir we were taught this code in class and I wasn't aware this was from a particular website. My apologies. I have closed this pull request

**Pull request closed without merging**